src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -58,7 +68,8 @@ interface IBlazor {
     loadSatelliteAssemblies?: any;
     sendJSDataStream?: (data: any, streamId: number, chunkSize: number) => void;
     getJSDataStreamChunk?: (data: any, position: number, chunkSize: number) => Promise<Uint8Array>;
-    receiveDotNetDataStream?: (streamId: number, data: any, bytesRead: number, errorMessage: string) => void;
+    receiveWebAssemblyDotNetDataStream?: (streamId: number, data: any, bytesRead: number, errorMessage: string) => void;
+    receiveWebViewDotNetDataStream?: (streamId: number, data: any, bytesRead: number, errorMessage: string) => void;
```

Comment:
`receiveDotNetDataStream` is unique in that it needs to know which `ICallDispatcher` initiated the call. So, I've added these two methods, and each of them passes in its respective `ICallDispatcher` instance to `receiveDotNetDataStream`. I'm trying to avoid having any platform-agnostic APIs that can't handle multiple platforms being available at the same time.